src/redux/store/store.ts

```diff
@@ -14,7 +14,7 @@ import { Env } from "../../common/environment";
 export default function createReduxStore(
     initialState?: IApplicationState,
     useLocalStorage: boolean = false): Store {
-    const paths: string[] = ["appSettings", "connections", "recentProjects"];
+    const paths: string[] = ["appSettings", "connections", "recentProjects", "auth"];
```

Comment:
React-oriented question: why we do not have "currentProjects" and "appError" in the paths?...

... whereas we have them defined in https://github.com/Cortexia-CH/VoTT/pull/1/files#diff-9e3972d5d9a1621a15d4d82b2548c5a2R11

Reply:
Those paths are used to create initial local storage when the app is running. I think that "currentProjects" and "appError" are not in those paths because maybe it's unnecessary to keep "appError" and "currentProjects" in the local storage. (maybe they're setup in other place) We've got "appError" and "currentProjects" in the redux store so it might be enough.

```suggestion
    const localStorage = require("../middleware/localStorage");
    const storage = localStorage.createLocalStorage({paths});
[...]
    const defaultState: IApplicationState = initialState;
    const store = createReduxStore(defaultState, true);
```